pkg/controllers/job/job_controller.go

```diff
@@ -127,53 +122,18 @@ func NewJobController(config *rest.Config) *Controller {
 	cc.jobSynced = cc.jobInformer.Informer().HasSynced
 
 	cc.cmdInformer = vkinfoext.NewSharedInformerFactory(cc.vkClients, 0).Bus().V1alpha1().Commands()
-	cc.cmdInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
-		FilterFunc: func(obj interface{}) bool {
-			switch t := obj.(type) {
-			case *v1corev1.Command:
-				return helpers.ControlledBy(t, helpers.JobKind)
-			case cache.DeletedFinalStateUnknown:
-				if cmd, ok := t.Obj.(*v1corev1.Command); ok {
-					return helpers.ControlledBy(cmd, helpers.JobKind)
-				}
-				runtime.HandleError(fmt.Errorf("unable to convert object %T to *v1.Command", obj))
-				return false
-			default:
-				runtime.HandleError(fmt.Errorf("unable to handle object %T", obj))
-				return false
-			}
-		},
-		Handler: cache.ResourceEventHandlerFuncs{
-			AddFunc: cc.addCommand,
-		},
+	cc.cmdInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: cc.addCommand,
```

Comment:
Can we keep the ``ControllerdBy`` filter logic here?

Reply:
Is there any command resource not managed by volcano?

Reply:
which is faster? Just asking.

Reply:
No for now.